### PR TITLE
chore: Add GitHub release badge to the repository

### PR DESCRIPTION
**Pull request by electrosenpai**
This PR creates a new branch `release-badge` and commits changes to the `README.md`, adding the GitHub release badge for better visibility of project releases.

### CODE REVIEW

**Review by vincent-pochet — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -78,6 +78,25 @@ Lago launched its v0.1 on June 2nd, 2022. Lots of new features are coming, and a
...
+| **lago-java-client**     | [![lago-java-client Release](https://img.shields.io/github/v/release/getlago/lago-java-client)](https://github.com/getlago/lago-java-client/releases) |
+| **Lago JavaScript Client**     | [![Lago JavaScript Client Release](https://img.shields.io/github/v/release/getlago/lago-javascript-client)](https://github.com/getlago/lago-javascript-client/releases) |
+| **Lago OpenAPI**     | [![Lago OpenAPI Release](https://img.shields.io/github/v/release/getlago/lago-openapi)](https://github.com/getlago/lago-openapi/releases) |
+| **OpenAPIClient-php**     | [![OpenAPIClient-php Release](https://img.shields.io/github/v/release/getlago/lago-php-client)](https://github.com/getlago/lago-php-client/releases) |
+| **Lago Python Client**     | [![Lago Python Client Release](https://img.shields.io/github/v/release/getlago/lago-python-client)](https://github.com/getlago/lago-python-client/releases) |
+| **Lago Ruby**     | [![Lago Ruby Release](https://img.shields.io/github/v/release/getlago/lago-ruby)](https://github.com/getlago/lago-ruby/releases) |
```
Comment (vincent-pochet) [inline README.md]: This repo is not maintained, we should archive it. So, no need to have it here.

File: README.md [hunk 2]
```diff
@@ -78,6 +78,25 @@ Lago launched its v0.1 on June 2nd, 2022. Lots of new features are coming, and a
...
+| **Lago Ruby Client**     | [![Lago Ruby Client Release](https://img.shields.io/github/v/release/getlago/lago-ruby-client)](https://github.com/getlago/lago-ruby-client/releases) |
+| **Rust API client for openapi**     | [![Rust API client for openapi Release](https://img.shields.io/github/v/release/getlago/lago-rust-client)](https://github.com/getlago/lago-rust-client/releases) |
```
Comment (vincent-pochet) [inline README.md]: This one is not officially maintained, so I think we should not list it here

File: README.md [hunk 3]
```diff
@@ -78,6 +78,25 @@ Lago launched its v0.1 on June 2nd, 2022. Lots of new features are coming, and a
...
+|--------------------|-----------------------------------------------------------------------------------------------------|
+| **Lago**           | [![Lago Release](https://img.shields.io/github/v/release/getlago/lago)](https://github.com/getlago/lago/releases) |
+| **Lago API**     | [![Lago API Release](https://img.shields.io/github/v/release/getlago/lago-api)](https://github.com/getlago/lago-api/releases) |
+| **Lago front**     | [![Lago front Testing Release](https://img.shields.io/github/v/release/getlago/lago-front)](https://github.com/getlago/lago-front/releases) |
+| **Lago Go Client**     | [![Lago Go Client Testing Release](https://img.shields.io/github/v/release/getlago/lago-go-client)](https://github.com/getlago/lago-go-client/releases) |
+| **lago-gotenberg**     | [![lago-gotenberg Release](https://img.shields.io/github/v/release/getlago/lago-gotenberg)](https://github.com/getlago/lago-gotenberg/releases) |
+| **lago-java-client**     | [![lago-java-client Release](https://img.shields.io/github/v/release/getlago/lago-java-client)](https://github.com/getlago/lago-java-client/releases) |
```
Comment (vincent-pochet) [inline README.md]: As for the rust client, the Java one is not officially maintained

**Review by vincent-pochet — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (electrosenpai) [reply README.md]: Roger ! Making all these changes :)

**Pull request merged**